packages/components/organisms/f-checkout/src/components/Selector.vue

```diff
@@ -103,6 +105,7 @@ export default {
         selectionChanged (selectedFulfilmentTime) {
             this.selectedAvailableFulfilmentTime = selectedFulfilmentTime;
             this.updateChangedField('orderTime');
+            this.updateHasAsapSelected(false);
```

Comment:
Is this correct? What if the user changes from ASAP to a pre-order time, and then switch back to ASAP? Wouldn't `hasAsapSelected` remain as `false`?

Reply:
Fixed.